llvm/lib/Target/AVR/MCTargetDesc/AVRAsmBackend.cpp

```diff
@@ -45,12 +49,18 @@ static void signed_width(unsigned Width, uint64_t Value,
 
     Ctx->reportError(Fixup.getLoc(), Diagnostic);
   }
+
+  return false;
 }
 
-static void unsigned_width(unsigned Width, uint64_t Value,
-                           std::string Description, const MCFixup &Fixup,
-                           MCContext *Ctx) {
-  if (!isUIntN(Width, Value)) {
+static bool checkUnsignedWidth(unsigned Width, uint64_t Value,
+                               std::string Description, const MCFixup &Fixup,
```

Comment:
This probably should not be std::string. Twine maybe?

Reply:
Sounds good, done!

Reply:
This is still std::string?

Reply:
Ah, I've adjusted the the inner code to use `Twine`s, but forgot about the parameter - changed.